code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -56,7 +56,14 @@
 	..()
 	reagents.add_reagent("synaptizine", 30)
 
+obj/item/weapon/reagent_containers/hypospray/combat/adminordrazine
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with adminordrazine for rapid healing."
+	volume = 100
 
+/obj/item/weapon/reagent_containers/hypospray/combat/adminordrazine/New()
+	..()
+	reagents.remove_reagent("synaptizine", 30)
+	reagents.add_reagent("adminordrazine", 100)

```

Comment:
I'm not sure how I feel about this. 100 units of adminordrazine is pretty ridiculous, but then again so are most other ERT items, so...